client/me/connected-application-item/index.jsx

```diff
@@ -60,19 +61,18 @@ module.exports = React.createClass( {
 	},
 
 	renderAccessScopeBadge: function() {
-		var meta = '',
-			connection = this.props.connection;
+		var meta = '';
```

Comment:
That's interesting, as I've not heard anyone say that - what's the benefit to knowing that it has been passed through a prop? Or, what's the disadvantage of having that declaration at the top of the current function (to indicate where it came from)?